pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -48,6 +48,13 @@ type TFJobSpec struct {
 	// Default to Running.
 	CleanPodPolicy *CleanPodPolicy `json:"cleanPodPolicy,omitempty"`
 
+	// TTLSecondsAfterFinishing is the TTL to clean up tf-jobs (temporary
+	// before kubernetes adds the cleanup controller).
+	// It may take extra ReconcilePeriod seconds for the cleanup, since
+	// reconcile gets called periodically.
+	// Default to infinite.
+	TTLSecondsAfterFinishing *int32 `json:"ttlSecondsAfterFinishing,omitempty"`
```

Comment:
nit: Rename to "TTLSecondsAfterFinished" to match 
https://docs.google.com/document/d/1U6h1DrRJNuQlL2_FYY_FdkQhgtTRn1kEylEOHRoESTc/edit?ts=5b58ac5b

Reply:
done